lessons/beginners/testing/index.md

```diff
@@ -133,14 +133,19 @@ test_secteni.py::test_secti ␛[32mPASSED␛[0m
 ␛[32m============= 1 passed in 0.00 seconds =============␛[0m
 ```
 
-Tento příkaz projde všechny soubory v aktuálním
-adresáři, jejichž jméno začíná na `test_`, zavolá v nich všechny funkce,
+Tento příkaz projde zadaný soubor, zavolá v něm všechny funkce,
 jejichž jméno začíná na `test_`, a ověří, že nevyvolají žádnou
-výjimku — typicky výjimku z příkazu `assert`.
+výjimku – typicky výjimku z příkazu `assert`.
 Pokud výjimka nastane, dá to `pytest` velice červeně
 najevo a přidá několik informací, které můžou
 usnadnit nalezení a opravu chyby.
 
+> [note]
+> Argument s názvem souboru můžeme vynechat: `python -m pytest -v`
```

Comment:
Tady chybí interpunkce.

Reply:
Já to chápal tak, že `delší příkaz` v tomto případě jako takový zastupuje větu. Ta předchozí končí dvojtečkou, tato koncem příkazu, protože za bashovými příkazy se tečka nepíše. Když jsem ji tam měl, přišlo mi to spíše divnější.

Reply:
ok